README.md

```diff
@@ -1,3 +1,44 @@
 # TheSportsDB
 GraphQL wrapper for TheSportsDB
 API doc - https://www.thesportsdb.com/api.php
+
+## How to use
+
+### Execute loal server
+
+Two ways for execute local server
+
+#### Using [Visual Studio Code](https://azure.microsoft.com/en-us/products/visual-studio-code/)
+
+Modify configurations of `launch.json`.
```

Comment:
Vscode debug config is already in place, no need for any modification.

Reply:
Sorry, this is my problem of PATH settings. Fixed in [67d2e2f](https://github.com/jsBlackBelt/TheSportsDB/pull/2/commits/67d2e2f5506ea82bd137cb35bdd5e9f008303969)